.travis.yml

```diff
@@ -15,5 +15,5 @@ cache:
 notifications:
   email: false
   webhooks:
-    secure: "FbvroC2dU48lfDbGBc+EiNFWV5npRgo4NWOnKtCZ1b0fsHUu25Ir92NoEwl2LkqOhHLi7llle/66ZFv+Hpy3ODZQyhWH58nZByMJRAdBCDqUb6wp3ztnwMzTwhJjWAWlja42m8VwG46PyPLeQsWEY5H6IyQtTNBLv9TgAddFkEXlYL9GHxCOkJ9XHIEqyIYmOZnj17uZXQ2qgG01iCXVjV846bV8mwuypGFjBth8UU4L3vWkcFusbznE6rs7Wbu4HvOtPFhO2tEwxmVRsQLn8KNzMvNR0MsRjRtRtRCmJTDmBdFYjOjlJ6NC24GUNk0lMSh+KNycIy61pYdqK/bGqzKAh8Jdob3ouojjPYF3usgtUmeB+ZEXN/IrM4nQlFXyHojm74juS1ljgVwas0gC9982kskTewtbEomtixU3mOKfDh4SdYIVw//LCYvV0n4qH0jgajh2mTsIAy6nAmBlmZkXDcjXBBlBOB+NCFClVBfw2NB8S3/LUtMIzy2yAP9P5clXWMZBjlKtrFisblLvF6S77QcBugkuCQ6tFIiOdfeoiGvudIa1o0xY1xJBNyPIzwSdKFs0g0kjMS5jclpjkanR1oo/TT0x1kaoDYPjNd2GHdTPg9tR0eKGs7LeqpfBfksz7tQH+cE0b7aipw4+yBb2PpphpSUwhSPNIFY38T0="
+    - secure: "c3JXHF4HUXGvyB/bCcMsU6943r1Os9ZpTJ+cPDxxhFW/U+XB+juhTJyD2FUDjHCzIoXxeR/N71zPOwYFR1swfPmMIpwxyDero0GlR/Z8l3UNcM+BXJyALCXLnJuk6Y08c5uYovXk1EiKixkBerOdmIbuWQR36hSvtAmvtawrCtDqXVh4IEOsdjYfZYHYT8HjE0mD6QZRmLsQlLfPzMAY7Crtg4oChX5FC19EyCeUMH+GK1SyHzIUQQUQC9QEU3f3fKMyESToN7MuV2nqlfSO0O3mknDRI+z1BfUfeRjz+lOK7K0x4K6SpAbLHIqaLYuHyrmZvi9vZq1tg9wFEnvMYcCz3Bw+/iBdQLdX9oSRiMQde94+NCtuPT5LDj4H4+kKLZmWp9zCSEvdXlK3DaRfhWPeIpuJMZgVHj/b+0TCSUrcPSSadntzF19/tTaI6ewKSev3HM32jImpST8WsEDSwJ0RWhObQnCtli7RfiMG2PCdnZqp/pWFDnRURSmXmThWdV6cIFsM1AN4biy4OzfV5rVVerjTlWxJoP41b0m26spqgDC0ARrnd2IgTArITF9USOrmyKZlk15iNWPzFxWsqYu2/gVvQA/c3vjKLA4MUnJg2TxZgv5lIVWIvfREeJCwPYkxW0q+PIRdcVWnpAtyOc/A44fI4S1oMeExSAOiel0=" 
```

Comment:
what is this webhook?

Reply:
I regenerated the one documented on the Build page, in case it would have solved the issue with Burgr not displaying start badge. But it was not the case. Do you prefer I revert this change?

Reply:
This webhook was outdated, I've updated the url on the build page, please re-encrypt with the new one.

Reply:
Done